test/module/shared_model/backend_proto/CMakeLists.txt

```diff
@@ -113,3 +112,13 @@ target_link_libraries(proto_block_factory_test
     shared_model_proto_backend
     shared_model_stateless_validation
     )
+
+if (IROHA_ROOT_PROJECT)
```

Comment:
Why it depends on root project?

Reply:
Because they use internal part of `shared_model_proto_backend`, which, in turn, depends on root project. You can find it in "shared_model/backend/protobuf/CMakeLists.txt", line 47